cmd/soroban-cli/src/commands/network/container/start.rs

```diff
@@ -164,40 +180,42 @@ impl Cmd {
     }
 
     fn container_name(&self) -> Name {
-        Name(self.name.clone().unwrap_or(self.network.to_string()))
+        Name(
+            self.args
+                .name
+                .clone()
+                .unwrap_or(self.args.network.to_string()),
+        )
     }
 
-    fn print_log_message(&self) {
-        let log_message = format!(
-            "ℹ️ To see the logs for this container run: stellar network container logs {container_name} {additional_flags}",
-            container_name = self.container_name().get_external_container_name(),
-            additional_flags = self.container_args.get_additional_flags(),
-        );
-        println!("{log_message}");
-    }
+    fn print_instructions(&self) {
+        let container_name = self.container_name().get_external_container_name().clone();
+        let additional_flags = self.args.container_args.get_additional_flags().clone();
+        let tail = format!("{container_name} {additional_flags}");
+
+        self.print.searchln(format!(
+            "You can see logs by running `stellar network container logs {}`",
```

Comment:
The 'you' is unnecessary:
```suggestion
            "Watch logs with `stellar network container logs {}`",
```